service/history/replication/stream.go

```diff
@@ -71,7 +71,7 @@ func WrapEventLoop(
 ) {
 	defer streamStopper()
 
-	for retryCount := 0; retryCount < 11; retryCount++ {
+	for {
```

Comment:
Is this change intentional?

Reply:
Yes it is. This change is to not disconnect the grpc stream if it is `ReplicationServiceError`. Initially it is infinite loop here. I added the retry counts at previous version just in case we miss categorize the error. But we learned from last Friday's incident that retry count here will still disconnect the stream and cause issue on network layer.

So I added additional metrics and planing to add alarm on them, so in case we miss category the error, we still get notified.